mlrun/runtimes/databricks_runtime_wrap.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+def run_mlrun_databricks_job(
+    context,
+    internal_handler,
+    is_local_code=True,
+    token_key="DATABRICKS_TOKEN",
+    **kwargs,
+):
+    import ast
+    import datetime
+    import json
+    import os
+    import shutil
+    import tempfile
+    import uuid
+
+    import astor
+    from databricks.sdk import WorkspaceClient
+    from databricks.sdk.service.compute import ClusterSpec
+    from databricks.sdk.service.jobs import Run, SparkPythonTask, SubmitTask
+
+    import mlrun
+
+    project_name = context.project
+    project = mlrun.get_or_create_project(project_name)
+    get_secret_func = project.get_secret
```

Comment:
Since this is client-code, then you don't need a specific provider to retrieve project-secrets. Project secrets will already be mounted as env variables and directly accessible to the `get_secret_or_env()` function. So, this can be removed.